### PR TITLE
Dm/fixes

### PR DESCRIPTION
**Pull request by dbmeneses**

### CODE REVIEW

File: src/main/java/org/sonarlint/intellij/ui/nodes/LabelNode.java [hunk 1]
```diff
@@ -21,6 +21,7 @@
 
 import com.intellij.openapi.wm.impl.welcomeScreen.BottomLineBorder;
 import com.intellij.ui.SimpleTextAttributes;
+import com.intellij.util.ui.JBUI;
 import java.awt.Insets;
```
Comment (SonarTech) [inline src/main/java/org/sonarlint/intellij/ui/nodes/LabelNode.java]: MINOR Remove this unused import 'java.awt.Insets'. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/main/java/org/sonarlint/intellij/ui/nodes/LocationNode.java [hunk 2]
```diff
@@ -22,6 +22,7 @@
 import com.intellij.openapi.editor.Document;
 import com.intellij.openapi.editor.RangeMarker;
 import com.intellij.ui.SimpleTextAttributes;
+import com.intellij.util.ui.JBUI;
 import java.awt.Insets;
```
Comment (SonarTech) [inline src/main/java/org/sonarlint/intellij/ui/nodes/LocationNode.java]: MINOR Remove this unused import 'java.awt.Insets'. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by SonarTech** [comment]
SonarQube analysis reported 6 issues
* ![MINOR][MINOR] 6 minor

Watch the comments in this conversation to review them.

#### 4 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [SonarQubeServerMgmtPanel.java#L51](https://github.com/SonarSource/sonarlint-intellij/blob/c57768041242eabbb95220f60d212366b5a96be7/src/main/java/org/sonarlint/intellij/config/global/SonarQubeServerMgmtPanel.java#L51): Remove this unused import 'java.awt.Insets'. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)
1. ![MINOR][MINOR] [SonarLintProjectBindPanel.java#L46](https://github.com/SonarSource/sonarlint-intellij/blob/c57768041242eabbb95220f60d212366b5a96be7/src/main/java/org/sonarlint/intellij/config/project/SonarLintProjectBindPanel.java#L46): Remove this unused import 'java.awt.Insets'. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)
1. ![MINOR][MINOR] [AutoTriggerStatusPanel.java#L40](https://github.com/SonarSource/sonarlint-intellij/blob/c57768041242eabbb95220f60d212366b5a96be7/src/main/java/org/sonarlint/intellij/ui/AutoTriggerStatusPanel.java#L40): Remove this unused import 'java.awt.Insets'. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)
1. ![MINOR][MINOR] [SonarLintAnalysisResultsPanel.java#L35](https://github.com/SonarSource/sonarlint-intellij/blob/c57768041242eabbb95220f60d212366b5a96be7/src/main/java/org/sonarlint/intellij/ui/SonarLintAnalysisResultsPanel.java#L35): Remove this unused import 'java.awt.Insets'. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: build.gradle [hunk 3]
```diff
@@ -29,13 +29,10 @@ intellij {
   if (project.hasProperty("ijVersion")) {
     version ijVersion
   } else {
-    version '2016.3.2'
+    version '2016.1.4'
```
Comment (janos-ss) [inline build.gradle]: Changing the value from `2016.3.2` down to `2016.1.4`... Why that version is better, and how is it an appropriate choice?

**Comment by SonarTech** [comment]
SonarQube analysis reported 6 issues
* ![MINOR][MINOR] 6 minor

Watch the comments in this conversation to review them.

#### 4 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [SonarQubeServerMgmtPanel.java#L51](https://github.com/SonarSource/sonarlint-intellij/blob/d3998926c215725849f92591843a7e6d0981ac23/src/main/java/org/sonarlint/intellij/config/global/SonarQubeServerMgmtPanel.java#L51): Remove this unused import 'java.awt.Insets'. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)
1. ![MINOR][MINOR] [SonarLintProjectBindPanel.java#L46](https://github.com/SonarSource/sonarlint-intellij/blob/d3998926c215725849f92591843a7e6d0981ac23/src/main/java/org/sonarlint/intellij/config/project/SonarLintProjectBindPanel.java#L46): Remove this unused import 'java.awt.Insets'. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)
1. ![MINOR][MINOR] [AutoTriggerStatusPanel.java#L40](https://github.com/SonarSource/sonarlint-intellij/blob/d3998926c215725849f92591843a7e6d0981ac23/src/main/java/org/sonarlint/intellij/ui/AutoTriggerStatusPanel.java#L40): Remove this unused import 'java.awt.Insets'. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)
1. ![MINOR][MINOR] [SonarLintAnalysisResultsPanel.java#L35](https://github.com/SonarSource/sonarlint-intellij/blob/d3998926c215725849f92591843a7e6d0981ac23/src/main/java/org/sonarlint/intellij/ui/SonarLintAnalysisResultsPanel.java#L35): Remove this unused import 'java.awt.Insets'. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: build.gradle (hunk 3 above)
Comment (dbmeneses) [reply build.gradle]: Well spotted - this is a mistake, I though I had changed it back.

**Pull request merged**